src/index.ts

```diff
@@ -14,49 +14,60 @@ import type Koa from 'koa';
 // the `koa-bodyparser` types "polyfill" the `koa` types
 import type * as _ from 'koa-bodyparser';
 
-export interface KoaContextFunctionArgument {
-  ctx: Koa.Context;
+export interface KoaContextFunctionArgument<
+  StateT = Koa.DefaultState,
+  ContextT = Koa.DefaultContext,
```

Comment:
Without the default type parameters, it would technically be a breaking change, as this type is `export`'ed.